docs/source/quickstart.md

```diff
@@ -33,11 +33,16 @@ Before installing JupyterHub, you will need:
 - TLS certificate and key for HTTPS communication
 - Domain name
 
+[nodesource]: https://github.com/nodesource/distributions#table-of-contents
+
 Before running the single-user notebook servers (which may be on the same
 system as the Hub or not), you will need:
 
 - [Jupyter Notebook](https://jupyter.readthedocs.io/en/latest/install.html)
-  version 4 or greater
+  version 4 or greater,
+  or [JupyterLab][] 3 or greater.
+
+[jupyterlab]: https://jupterlab.readthedocs.io
```

Comment:
```suggestion
[jupyterlab]: https://jupyterlab.readthedocs.io
```
Small typo.

I recently updated the README: https://github.com/jupyterhub/jupyterhub/pull/3601/files and put JupyterLab first to emphasize it a bit more, worth doing the same here?

Reply:
Good call, updated.